mobilecoind-json/tests/test.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+
+# check for unexpected ProcessedTxOut data when using mobilecoind-json
+
+# usage: ./test.sh <sender master key as 64 hex chars>
+
+# TODO: Check that mobilecoind and mobilecoind-json are both running and start as needed
+
+# set up a monitor for the sender account
+ENTROPY=`echo $1 | sed -n '/^[[:xdigit:]]\{64\}/p'`
+
+# check if the master key is valid
+if [ -z "$ENTROPY" ];
+  then
+    echo "You must supply a valid 64 hex character master key for the sender."
+    exit 0
+fi
+
+ACCOUNT_KEY=`curl -s localhost:9090/entropy/${ENTROPY}`
+
+echo "Sending a test payment for master key = $ENTROPY"
+
+MONITOR_ID=`curl -s localhost:9090/monitors \
+  -d '{"account_key": '"$ACCOUNT_KEY"', "first_subaddress": 0, "num_subaddresses": 10}' \
+  -X POST -H 'Content-Type: application/json' \
+  | jq -r '.monitor_id'`
+
+# wait until the ledger is current
```

Comment:
I created a ticket to expose the API for `GetNetworkStatus` so that we can do this properly. As it is now, the transaction is likely to fail since the monitor will outrun the ledger download and the script will continue without utxos to spend.